src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -2928,6 +2809,19 @@ public void var4(InputType inputType, ReaderType readerType, TransformType trans
         }
     }
 
+    internal sealed class AllowDefaultResolverContext : IDisposable
+    {
+        public AllowDefaultResolverContext()
+        {
+            AppContext.SetSwitch("Switch.System.Xml.AllowDefaultResolver", true);
+        }
+
+        public void Dispose()
+        {
+            AppContext.SetSwitch("Switch.System.Xml.AllowDefaultResolver", false);
+        }
```

Comment:
Consider putting the switch name into a const so we only need to name it once, e.g.
```C#
internal sealed class AllowDefaultResolverContext : IDisposable
{
    private const string SwitchName = "Switch.System.Xml.AllowDefaultResolver";
    public AllowDefaultResolverContext() => AppContext.SetSwitch(SwitchName, isEnabled:true);
    public void Dispose() => AppContext.SetSwitch(SwitchName, isEnabled:false);
}
```